pkg/metrics-server/server.go

```diff
@@ -129,40 +128,57 @@ func (ms *MetricsServer) scrape(ctx context.Context, startTime time.Time) {
 	recvErr := ms.storage.Store(data)
 	if recvErr != nil {
 		klog.Errorf("unable to save metrics: %v", recvErr)
-
-		// any failure to save means we're unhealthy
-		healthyTick = false
+		// any failure to save means unhealthy scrape
+		healthyScrape = false
 	}
 
 	collectTime := time.Since(startTime)
 	tickDuration.Observe(float64(collectTime) / float64(time.Second))
 	klog.V(6).Infof("...Cycle complete")
 
 	ms.healthMu.Lock()
-	ms.lastOk = healthyTick
+	ms.lastOk = true
```

Comment:
I don't see place where we would set lastOk to false. What's the intention here?